src/main/java/com/salesforce/storm/spout/dynamic/SpoutCoordinator.java

```diff
@@ -146,14 +152,14 @@ public boolean hasVirtualSpout(final VirtualSpoutIdentifier spoutIdentifier) {
      * Open the coordinator and begin spinning up virtual spout threads.
      * @param topologyConfig topology configuration.
      */
-    public void open(Map<String, Object> topologyConfig) {
+    public void open(final Map<String, Object> config) {
```

Comment:
Renamed this argument because it was shadowing the class property topologyConfig, which could be error prone.